scanner/src/main/kotlin/scanners/scanoss/ScanOssConfig.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2022 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.scanner.scanners.scanoss
+
+import java.security.SecureRandom
+import java.security.cert.X509Certificate
+
+import javax.net.ssl.HostnameVerifier
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManager
+import javax.net.ssl.X509TrustManager
+
+import okhttp3.OkHttpClient
+
+import org.ossreviewtoolkit.clients.fossid.FossIdServiceWithVersion
+import org.ossreviewtoolkit.clients.scanoss.ScanossService
+import org.ossreviewtoolkit.model.config.ScannerConfiguration
+import org.ossreviewtoolkit.utils.core.OkHttpClientHelper
+import org.ossreviewtoolkit.utils.core.log
+
+/**
+ * A data class that holds the configuration options supported by the [ScanOss] scanner. An instance of this class is
+ * created from the options contained in a [ScannerConfiguration] object under the key _ScanOss_. It offers the
+ * following configuration options:
+ */
+internal data class ScanOssConfig(
+    /** URL of the ScanOSS API server. */
+    val apiURL: String,
+
+    /** API Key required to authenticate with the ScanOSS API server. */
+    val apiKey: String
+) {
+    companion object {
+        /** Name of the configuration property for the API URL. */
+        private const val API_URL_PROPERTY = "apiURL"
+
+        /** Name of the configuration property for the API key. */
+        private const val API_KEY_PROPERTY = "apiKey"
+
+        fun create(scannerConfig: ScannerConfiguration): ScanOssConfig {
+            val scanOssIdScannerOptions = scannerConfig.options?.get("ScanOss")
+
+            requireNotNull(scanOssIdScannerOptions) { "No ScanOSS Scanner configuration found." }
+
+            val apiURL = scanOssIdScannerOptions[API_URL_PROPERTY] ?: "https://osskb.org/api/"
+            val apiKey = scanOssIdScannerOptions[API_KEY_PROPERTY].orEmpty()
+
+            return ScanOssConfig(apiURL, apiKey)
+        }
+    }
+
+    /**
+     * Create the [FossIdServiceWithVersion] to interact with the FossID instance described by this configuration.
+     */
+    fun createService(): ScanossService {
```

Comment:
I realized only just now that the service is called `ScanossService`, but in all other classes we seem to be using `ScanOss...`. Could you please add a commit to align this?